instance/static/js/src/instance.js

```diff
@@ -175,6 +178,35 @@ app.controller("Details", ['$scope', '$state', '$stateParams', 'OpenCraftAPI',
             });
         };
 
+        $scope.editNotes = function() {
+            let data = {"notes": DOMPurify.sanitize($scope.instance.notes)};
```

Comment:
We want to sanitize the output, not the input, since we're doing it on the frontend - this sanitizing should be part of the function that renders the markdown text to html.

Reply:
Shouldn't we also sanitize the content for the backend to avoid SQL injections? Or is it not necessary?

Reply:
If we want to sanitize input, then we should do that on the backend - frontend sanitation is easily bypassed.